bin/i18n/sync-codeorg-out.rb

```diff
@@ -31,6 +34,80 @@ def rename_from_crowdin_name_to_locale
   end
 end
 
+# Distribute downloaded translations from i18n/locales
+# back to blockly, apps, pegasus, and dashboard.
+def distribute_translations
```

Comment:
Would be good to split up this huge method.

The Dashboard, Apps, Blockly Core, and Pegasus steps have similar structure — can we extract a common helper method?

Reply:
looking over this closer, it feels like the main thing these all share in common is the special-casing for Armenian, which I'm pretty sure I'm gonna be able to remove (in a separate PR).

I vote we leave this as-is for now, and when I figure out whether or not I can stop special-casing Armenian I'll improve this then